### ISSUE TITLE
Scout Double Jump Convar Not Working

### ISSUE THREAD

**Issue by derekstephen** [Scags/TF2-Jailbreak-Redux#71]
Hi, I am trying to allow scout double jump, but when I set the convar to allow it scouts still can't double jump. The same goes for Pyro Airblast and Demo Charge.

There are no errors when the server runs. Any ideas?

**Comment by MrPanica** [comment]
Try it => tf_scout_air_dash_count 2

**Comment by derekstephen** [comment]
> 
> 
> Try it => tf_scout_air_dash_count 2

That still doesn't fix the issue. I've also tried setting `sm_tf2jr_double_jump` to equal 0, 1, and 2 and it doesn't work. The same with `sm_tf2jr_airblast` and `sm_tf2jr_demo_charge` I've tried setting them to equal 0 and 1 and it won't work still to let them airblast or charge.

**Comment by MrPanica** [comment]
This is strange, everything is working fine for me. Variables are not applied immediately after their change, do you have them in the config?

**Comment by Scags** [comment]
Can you assure that 'TF2Attributes' is running on your server? Also if you're compiling yourself, do you have the 'TF2Attributes' include file?
Not sure what's up with demo charging and the ever-painful muting system.

**Comment by derekstephen** [comment]
> 
> 
> This is strange, everything is working fine for me. Variables are not applied immediately after their change, do you have them in the config?

Yes, I have it all the convars in my server.cfg. I've restarted the server when I tried changing everything, even tested switching maps.

I'm really not sure what could be causing it all. I don't have any other weird plugins that would modify or do something of this nature. If it keeps up, I might try rebuilding the server and see if the issue continues.
